woocommerce/woocommerce
# Added admin email config override

### Changes proposed in this Pull Request:

This PR adds a new administrator email configuration override.

### How to test the changes in this Pull Request:

1. Make sure CI passes
2. In your local environment, change the admin email address to a different value
3. Run the order emails test using the new email value and the newly added config override:

```bash
SMOKE_TEST_ADMIN_USER_EMAIL="some@email.com" UPDATE_WC=1 npx wc-e2e test:e2e-dev ./tests/e2e/specs/wp-admin/order-emails.test.js
```

4. Verify the tests pass.
